src/components/Text/native/Text.js

```diff
@@ -1,29 +1,38 @@
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { Platform } from 'react-native';
+
+const isPlatformAndroid = !(Platform.OS === 'ios');
+
+export const fontFamilies = {
+  normal: isPlatformAndroid ? 'Roboto' : 'SFProDisplay-Regular',
+  medium: isPlatformAndroid ? 'Roboto-Medium' : 'SFProDisplay-Medium',
+  semibold: isPlatformAndroid ? 'Roboto-Medium' : 'SFProDisplay-Semibold',
+  bold: 'Averta-Bold',
+};
 
 const Text = styled.Text`
-  ${(p) => p.color ? `color: ${p.theme.color[p.color]};` : ''}
-  ${(p) => p.size ? `font-size: ${p.theme.fontSize[p.size]};` : ''}
-  ${(p) => p.weight ? `font-weight: ${p.theme.fontWeight[p.weight]};` : ''}
-  ${(p) => {
-    if (p.family) {
-      return `font-family: ${p.family}`;
-    }
-    return `font-family: ${p.theme.fontFamily[p.weight]}`;
-  }}
-  `;
+  color: ${(props) => props.theme.color[props.color]};
+  font-size: ${(props) => props.theme.fontSize[props.size]};
+  font-weight: ${(props) => props.theme.fontWeight[props.weight]};
+  font-family: ${(props) => fontFamilies[props.family]};
```

Comment:
You can directly use the `props.weight` prop here instead of accepting another `props.family` prop.

This will allow you to omit the `family` prop completely, allowing you to decide the family based on the `weight` prop alone

Reply:
It makes sense, but in many places there are different values being used for both. What do you think I should do?